dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/CodeGenerateUtils.java

```diff
@@ -1,4 +1,6 @@
-/** Copyright 2010-2012 Twitter, Inc.*/
+/**
```

Comment:
This header looks unusual to me. Is it compatible with Apache 2.0?

Reply:
Since the origin code is copy from twitter, so need to announce it.